public/plugin.tsx

```diff
@@ -515,10 +515,10 @@ export class ObservabilityPlugin
 
     const renderCreateAccelerationFlyout = (
       selectedDatasource: string,
-      dataSourceMDSId?: string,
       databaseName?: string,
       tableName?: string,
-      handleRefresh?: () => void
+      handleRefresh?: () => void,
+      dataSourceMDSId?: string
```

Comment:
This order change might need a change in workbench? How do we know this is not causing a new regression, do we have test setup for this?

Reply:
yes same instance with the fix has it

Reply:
I meant some cypress tests or jest tests, to catch any regression in workbench. For example in workbench we are passing the params to this function: https://github.com/opensearch-project/dashboards-query-workbench/blob/888a8fe83b5e0a732866aaddb7a229eabed0f150/public/components/SQLPage/SQLPage.tsx#L71 Since, we do not have named params the MDSId will be mapped to `databaseName` now.